### PR TITLE
Adtelligent: ext aid type

### PR DESCRIPTION
**Pull request by dkornet-ad**

### CODE REVIEW

**Review by aradilov — COMMENTED** [review]
done

**Comment by GeneGenie** [comment]
changes adressing https://github.com/prebid/Prebid.js/issues/6361

**Comment by onkarvhanumante** [comment]
@dkornet-adtelligent 

PR tests are failing.  Requesting to fix them
```
--- FAIL: TestValidParams (0.01s)
    params_test.go:22: Schema rejected adtarget params: {"aid":"123"}
    params_test.go:22: Schema rejected adtarget params: {"aid":"123","siteId":0,"bidFloor":0}
FAIL
```

**Comment by dkornet-adtelligent** [comment]
@onkarvhanumante fixed

File: static/bidder-params/adtarget.json [hunk 1]
```diff
@@ -14,7 +14,10 @@
       "description": "An ID which identifies the site selling the impression"
     },
     "aid": {
-      "type": "integer",
+      "type": [
+        "integer",
+        "string"
+      ],
```
Comment (onkarvhanumante) [inline static/bidder-params/adtarget.json:20]: requesting to add json test where aid is string

File: static/bidder-params/adtelligent.json [hunk 2]
```diff
@@ -14,7 +14,10 @@
       "description": "An ID which identifies the site selling the impression"
     },
     "aid": {
-      "type": "integer",
+      "type": [
+        "integer",
+        "string"
```
Comment (onkarvhanumante) [inline static/bidder-params/adtelligent.json:19]: requesting to add json test where aid is string

File: static/bidder-params/openweb.json [hunk 3]
```diff
@@ -14,7 +14,10 @@
       "description": "An ID which identifies the site selling the impression"
     },
     "aid": {
-      "type": "integer",
+      "type": [
+        "integer",
+        "string"
```
Comment (onkarvhanumante) [inline static/bidder-params/openweb.json:19]: requesting to add json test where aid is string

File: static/bidder-params/adtarget.json (hunk 1 above)
Comment (dkornet-ad) [reply static/bidder-params/adtarget.json:20]: adapters/adtarget/params_test.go:45 adapters/adtarget/params_test.go:61

File: static/bidder-params/adtelligent.json (hunk 2 above)
Comment (dkornet-ad) [reply static/bidder-params/adtelligent.json:19]: adapters/adtelligent/params_test.go:45 adapters/adtelligent/params_test.go:61

File: static/bidder-params/openweb.json (hunk 3 above)
Comment (dkornet-ad) [reply static/bidder-params/openweb.json:19]: adapters/openweb/params_test.go:45 adapters/openweb/params_test.go:61

**Review by onkarvhanumante — APPROVED** [review]

**Comment by dkornet-ad** [comment]
@onkarvhanumante @Sonali-More-Xandr

**Review by Sonali-More-Xandr — APPROVED** [review]

**Comment by GeneGenie** [comment]
Hello, @onkarvhanumante  can this be merged ? =)

**Review by onkarvhanumante — APPROVED** [review]

**Comment by onkarvhanumante** [comment]
> Hello, @onkarvhanumante can this be merged ? =)

Thanks for keeping patience with PR reviews. PR is merged now.

**Pull request merged**